pom.client.xml

```diff
@@ -389,6 +389,10 @@
                 <title>Azure Storage - Blobs</title>
                 <packages>com.azure.storage.blob*</packages>
               </group>
+              <group>
+                <title>Azure Storage - Common</title>
+                <packages>com.azure.storage.common*</packages>
+              </group>
```

Comment:
Two comments:
1. I'm almost certain the answer is no, but it would be cool if we could have the common package listed under blobs, files, and queues. What happens if you list this package in multiple places (I'm guessing JavaDoc will choose the first instance and ignore all others).
2. Despite my preference for alphabetical ordering, I would always suggest putting common above the rest of the same service type, so common should go above blob, not beneath it.

Reply:
1. Using the same package name in different groups will fail to generate javadocs.
```
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-javadoc-plugin:3.0.1:jar (attach-javadocs) on project azure-core: MavenReportException: Error while generating Javadoc: 
[ERROR] Exit code: 1 - java.util.MissingResourceException: Can't find resource for bundle java.util.PropertyResourceBundle, key doclet.Same_package_name_used

```

2. Done